backend/lobby/consumers.py

```diff
@@ -101,8 +104,36 @@ def gamelog_send_handler(self, data):
 
         self.send_server_response_to_client(username, room, mud_gamelog_input)
 
+    def selected_card_handler(self, data):
+        username = data['user']
+        room = data['room']
+
+        print("selected card handler")
+
+        payload = data['payload']
+
+        self.send_server_response_to_client(username, room, payload)
+
+    def get_the_three_store_cards(self):
```

Comment:
Instead of using REST, you should use a channels message back to the client to transmit your store cards.

See 'server_response' as an example of sending to the client.

Send the array of cards from deck_handler.store back to the client. Array should be formatted appropriately so your CardStore from #149 will function as you intended.